shared/src/business/useCases/createCaseFromPaperInteractor.ts

```diff
@@ -102,7 +102,6 @@ export const createCaseFromPaperInteractor = async (
     { applicationContext },
   ).validate();
 
-  // invoke the createCase interactor
```

Comment:
Nice catch

Reply:
that's owed to those who made this change on staging :)